src/driver/cockroachdb/CockroachQueryRunner.ts

```diff
@@ -225,7 +225,7 @@ export class CockroachQueryRunner
         } else {
             this.storeQueries = false
             this.transactionDepth -= 1
-            await this.query("RELEASE SAVEPOINT cockroach_restart")
+            // await this.query("RELEASE SAVEPOINT cockroach_restart")
```

Comment:
If we have some commented out code, I think we should include some explanation of why it is commented out, or just have it deleted altogether.

Reply:
Added a comment about it and a link to this issue